Demo/HYPFormsTests/HYPFormTargetTests.m

```diff
@@ -29,6 +29,26 @@ - (void)testInitWithDictionary
     XCTAssertEqualObjects(target.actionTypeString, @"hide");
     XCTAssertTrue(target.type == HYPFormTargetTypeSection);
     XCTAssertTrue(target.actionType == HYPFormTargetActionHide);
+
+    target = [[HYPFormTarget alloc] initWithDictionary:@{@"id": @"first_name",
+                                                         @"type": @"field",
+                                                         @"action": @"disable"}];
+    XCTAssertNotNil(target);
+    XCTAssertEqualObjects(target.targetID, @"first_name");
+    XCTAssertEqualObjects(target.typeString, @"field");
+    XCTAssertEqualObjects(target.actionTypeString, @"disable");
+    XCTAssertTrue(target.type == HYPFormTargetTypeField);
+
+    XCTAssertTrue(target.actionType == HYPFormTargetActionDisable);
```

Comment:
Fixed in https://github.com/hyperoslo/HYPForms/commit/350d3c1ff57d2c2aca0963f4a147b72067bab7da